### PR TITLE
Describe MO authentication for the REST interface

### PR DESCRIPTION
**Pull request by ahaeber**

### CODE REVIEW

File: sections/interfaces/rest.md [hunk 1]
```diff
@@ -290,3 +290,50 @@ If the gateway does not receive a HTTP 200 acknowledge it will try resending the
...
+
+### Header with a specific value
+
+Adds a header to requests with a given name and value.
+
+A few examples how this method can be used:
+Header name | Header value | Description
+----------- | -------------| -------------------
+X-API-Key   | 3141592654   | For a fixed API-key
+Authenticaiton | Bearer <jwt> | For a fixed bearer token, where <jwt> is a [JSON Web Token (JWT)](https://tools.ietf.org/html/rfc7519).
```
Comment (svenstaleosa) [inline sections/interfaces/rest.md]: Authentication, small typo

**Review by svenstaleosa — CHANGES_REQUESTED** [review]
Should the header name in example be "Authorization"?

**Comment by ahaeber** [comment]
> Should the header name in example be "Authorization"?

Indeed, it should. Thanks.

File: sections/interfaces/rest.md (hunk 1 above)
Comment (ahaeber) [reply sections/interfaces/rest.md]: Changed to Authorization, ref your other comment.

**Pull request merged**